apps/mobile/src/queries/balance/sip10-balance.query.ts

```diff
@@ -12,9 +13,32 @@ export function useSip10TotalBalance() {
   return toFetchState(useSip10AggregateBalanceQuery(accounts.map(account => ({ account }))));
 }
 
-export function useSip10AccountBalance(fingerprint: string, accountIndex: number) {
+export function useSip10AccountBalance(
```

Comment:
Do we need to have a separation between `useSip10AccountBalance` and `useManagedSip10Tools`? Maybe they can be merged so that `useSip10AccountBalance` also can return `isEnabled`.